sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -155,6 +155,52 @@ public void deleteItem() throws Exception {
         assertThat(deleteResponse.getStatusCode()).isEqualTo(204);
     }
 
+    @Test(groups = {"simple"}, timeOut = TIMEOUT)
+    public void deleteAllItemsByPartitionKeyAsync() throws Exception {
```

Comment:
I don't see anything async in this method. any reason this is suffixed. with async?

if it is not testing async api, please remove the async suffix

Reply:
Removed the async suffix.